src/test/java/org/jongo/AggregateTest.java

```diff
@@ -37,6 +45,16 @@
 
 
     private MongoCollection collection;
+    private MongoCollection friendCollection;
+    private MongoCollection externalTypeCollection;
+    
+    @SuppressWarnings("serial")
+    public AggregateTest() {
```

Comment:
May be we can regroup mixins tests from SaveTest and AggregateTest into a dedicated MixinsTest